docs/framework/deployment/index.md

```diff
@@ -0,0 +1,18 @@
+# Installing the .NET Framework on Windows
+
+The .NET Framework comes with Windows. There are multiple versions of the .NET Framework, however, each version of Windows only includes one version. If you would like to use a different version of the .NET Framework for a given version of Windows, then you must install it.
+
+The installation requirements and experience for the .NET Framework differs by Windows and .NET Framework version. The following documents describe the differences that you need to know about for each Windows version.
+
+Window Server versions are not listed. You can click on the matching Windows client version below. The instructions will be the same.
```

Comment:
I agree. I think we need to list Server versions and call out those out in the individual topics too. We had similar complaints for this topic: https://docs.microsoft.com/en-us/dotnet/articles/core/rid-catalog#windows-rids